common-lib/src/main/scala/models/Subscription.scala

```diff
@@ -36,18 +40,37 @@ object Subscription {
 
   type Query = Map[String, Seq[String]]
 
-  def id(endpoint: SubscriptionEndpoint): String = {
+  val form = Form(
+    mapping("id" -> text)
+    (DeleteSubscription.apply)(DeleteSubscription.unapply)
```

Comment:
🤔 what's this doing?

Reply:
Just copy paste of how we're using the Play Forms DSL in the editorial support page. It hydrates a case class from the given form data.

Since we just have an ID I'll take a look and see if we don't need the case class.

Reply:
Reading the play docs it seems they encourage the use of case classes (with decent reasons for any forms that aren't as basic as mine). I'll leave it I think.

https://www.playframework.com/documentation/2.6.x/ScalaForms